plugins/CorePluginsAdmin/MarketplaceApiClient.php

```diff
@@ -123,9 +124,16 @@ public function searchForThemes($keywords, $query, $sort)
         return array();
     }
 
+    public static function getPiwikVersion()
+    {
+        return StaticContainer::get('marketplacePiwikVersion');
```

Comment:
This is probably not the best way to overwrite the piwik version immediately after the piwik files were updated but it will be only temporary there as we will refactor the marketplace in Piwik 3 or so